### PR TITLE
docs(readme): specify environment in code blocks

### PR DESCRIPTION
**Pull request by waynevanson**
This enables formatting to be present in `readme.md` for code blocks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,9 +253,9 @@ bootstrap({
 
 ### 3. Publish it to npm and use it!
 
-```
-npm install company-commit --save-dev
-./node_modules/.bin/company-commit
+```sh
+npm install company-commit --save-dev \
+  ./node_modules/.bin/company-commit
```
Comment (LinusU) [inline README.md:258]: I think that the trailing slash will make this get interpreted as: `npm install company-commit --save-dev ./node_modules/.bin/company-commit` by sh? It's intended to be two different commands...

Maybe move `--save-dev` and add a blank line to make it more obvious 🤔 

```suggestion
npm install --save-dev company-commit

./node_modules/.bin/company-commit
```

**Review by LinusU — APPROVED** [review]
Neat! Just one small thing, otherwise LGTM 🙌

File: README.md (hunk 1 above)
Comment (waynevanson) [reply README.md:258]: Sorry about that! I read it wrong. I'll revert my changes when I get home tonight

**Comment by waynevanson** [comment]
I was just about to change this.
Sorry about not getting back to you on this one and thank you for making the changes.

**Comment by dmwelch** [comment]
No worries

**Pull request merged**